### DISCUSSION TITLE
How to enable line pointer for legend indicator for pie chart

### DISCUSSION THREAD

**Discussion by SaravananArumugam24** [apexcharts/apexcharts.js#4609] [General]
Hi everyone,

I have added snapshots of canvas pie charts. We need similar legend indicator options for ApexCharts. Please let us know if such options are available.

![image](https://github.com/user-attachments/assets/9c9ed08b-a6d0-4411-bc3d-09e77e476dc8)

Thanks in advance

**Comment by SaravananArumugam24** [comment]
Hi ,

Is the mentioned feature available, or does it need to be developed?

**Comment by xylplm** [comment]
Yes, our project also needs it very much. When the node area is very small, the pointer will be very clear. I now have to introduce bloated echars to implement the project. I hope Apexcharts can achieve it

**Comment by syueya** [comment]
yes， also hope can have this options。

**Comment by mantou568** [comment]
+I have been waiting for this feature for a long time. Apexcharts has never had line indicators for pie charts, which is very inconvenient. Incomplete display of segmented elements, unable to display clearly.
We strongly recommend developers to add this feature. This is something that other statistical charts have.

**Reply by xylplm** [reply]
@junedchhipa
